OneSignalSDK/onesignal/src/main/java/com/onesignal/UserStatePushSynchronizer.java

```diff
@@ -63,6 +69,13 @@ void onSuccess(String responseStr) {
         }
     }
 
+    @Override
+    String getExternalId(boolean fromServer) {
+        synchronized(syncLock) {
+            return toSyncUserState.syncValues.optString("external_user_id", null);
```

Comment:
can "external_user_id" be on a constant?

Reply:
We dont use constants for any of our other keys. We should eventually migrate to do this though